routes/organisation.py

```diff
@@ -166,7 +166,7 @@ def get_organisations_namespaces():
 
 
 # Deleting an Organisations Namespace
-@organisation_bp.route('/delete/namespace', methods=['DELETE'])
+@organisation_bp.route('/delete/namespace', methods=['POST'])
```

Comment:
Wrong method. It should be `DELETE`.

Reply:
Also, you need to revise that endpoint to something more meaningful, say `organisations/<int:id>/namespaces/<int:id>` so as to follow the REST convention

Reply:
I just noticed you shall have to update all other endpoints to a similar fashion as in the above comment

Reply:
Thanks